### PR TITLE
Imagemapper color

### PR DESCRIPTION
**Pull request by sankhesh**
Use lookup table and color map presets for SliceRepresentation

### CODE REVIEW

File: usage/www/index.html [hunk 1]
```diff
@@ -2,19 +2,7 @@
...
-        <option>Geometry/PolyDataViewer</option>
-        <option>Geometry/PolyDataWithData</option>
-        <option>Geometry/ProcessingPipeline</option>
-        <option>Geometry/SourceViewer</option>
-        <option>Geometry/Glyph</option>
-        <option>Geometry/CutterExample</option>
         <option>Volume/SliceRendering</option>
-        <option>Volume/SyntheticVolumeRendering</option>
-        <option>Volume/VolumeRendering</option>
-        <option>Volume/DynamicUpdate</option>
```
Comment (jadh4v) [inline usage/www/index.html:17]: Is this change correct? Looks like you have removed these examples for testing, but forgot to re-enable them before commit?

**Comment by sankhesh** [comment]
@jourdain Please review and merge

File: src/core/SliceRepresentation.js [hunk 2]
```diff
@@ -111,8 +115,6 @@ export default class SliceRepresentation extends Component {
       changed = true;
       const preset = vtkColorMaps.getPresetByName(colorMapPreset);
       this.lookupTable.applyColorMap(preset);
-      this.lookupTable.setMappingRange(...colorDataRange);
-      this.lookupTable.updateRange();
     }
```
Comment (jourdain) [inline src/core/SliceRepresentation.js:118]: Should you keep the scaling on that colorDataRange?

File: src/core/View.js [hunk 3]
```diff
@@ -324,9 +324,9 @@ export default class View extends Component {
       visibility: false,
       dataBounds: [-1, 1, -1, 1, -1, 1],
     });
-    this.cubeAxes
-      .getActors()
-      .forEach(({ setVisibility }) => setVisibility(false));
+    Array.from(this.cubeAxes.getActors()).forEach(({ setVisibility }) =>
+      setVisibility(false)
+    );
     this.cubeAxes.setCamera(this.camera);
```
Comment (jourdain) [inline src/core/View.js:330]: Why do you need `Array.from()`

File: src/core/View.js [hunk 4]
```diff
@@ -506,9 +506,9 @@ export default class View extends Component {
     }
 
     if (this.cubeAxes.setVisibility(showCubeAxes)) {
-      this.cubeAxes
-        .getActors()
-        .forEach(({ setVisibility }) => setVisibility(showCubeAxes));
+      Array.from(this.cubeAxes.getActors()).forEach(({ setVisibility }) =>
+        setVisibility(showCubeAxes)
```
Comment (jourdain) [inline src/core/View.js:510]: Why `Array.from`?

File: src/core/SliceRepresentation.js (hunk 2 above)
Comment (sankhesh) [reply src/core/SliceRepresentation.js:118]: Nope - That happens here: https://github.com/Kitware/react-vtk-js/blob/e5d7e5f6fa934288c8f53adc3567bdf8a79d04a9/src/core/SliceRepresentation.js#L126-L146

File: src/core/View.js (hunk 3 above)
Comment (sankhesh) [reply src/core/View.js:330]: I was hitting an error where `getActors()` was returning `null` and `forEach` works on arrays. This change ensures that it is always an array.

File: src/core/View.js (hunk 4 above)
Comment (sankhesh) [reply src/core/View.js:510]: I was hitting an error where getActors() was returning null and forEach works on arrays. This change ensures that it is always an array.

**Pull request merged**